app/src/client/Main.css

```diff
@@ -37,6 +37,17 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
+/* Preload Material Symbols Outlined font */
```

Comment:
Interestimg, so this is a hack to get fonts downloaded immediately? What happens if do not do this?

Reply:
I am not sure if we have already tried it, but if not maybe we can also see if: `<link rel="preload" ...` helps